cns/api.go

```diff
@@ -140,7 +140,7 @@ type Response struct {
 	Message    string
 }
 
-// getNumberOfCPUCores describes reponse that returns the host local IP Address.
+// getNumberOfCPUCores describes reponse that returns num of cpu cores present on host.
```

Comment:
Shouldn't the comment say - NumOfCPUCoresResponse describes ... ?

Reply:
test

Reply:
why did you resolve this?